src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_VALUE = 25;
+    public static final int VALUE_FOR_DEFAULT_FIGURE = 10;
+
+    private Circle circle = new Circle(Color.WHITE.name(), VALUE_FOR_DEFAULT_FIGURE);
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(FigureClass.values().length);
+        switch (FigureClass.values()[index]) {
+            case CIRCLE:
+                return new Circle(random.nextInt(MAX_VALUE));
```

Comment:
```suggestion
            case CIRCLE:
                return getRandomCircle();
```
lets add private methods to get each random figure